src/device/usbd_control.c

```diff
@@ -55,8 +55,17 @@ typedef struct
 
 static usbd_control_xfer_t _ctrl_xfer;
 
+#if defined(TU_HAS_NO_ATTR_ALIGNED)
```

Comment:
As I understand it, there is no need for alignment if DMA is not used. And the current implementation does not use DMA.

If using DMA, the buffer must be 2 bytes aligned.

Reply:
RX MCU does not need an alignment on this. It was introduced, because there was an error in `dcd_usba.c`, which is corrected. On my test code the `_usbd_ctrl_buf` buffer started on an odd address and with the error in `dcd_usba.c` the transfer over an endpoint was faulty. So this fix is no longer required (as long as no DMA transfer is supported).

Reply:
It is surprised  that  ccrx has align pragma for function but not variable. Anyway there is several other places that may requires alignment especially endpoint buffer in each class driver. Therefore, maybe we revert this changes for now, and will revisit these later when doing DMA. SInce doing this alone is not enough anyway.